<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgColorPulsation</class>
 <widget class="QDialog" name="CQDlgColorPulsation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>211</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Emissive Color Component Pulsation</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QCheckBox" name="qqEnablePulsation">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>171</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Enable pulsation</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="qqRealTime">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Real-time</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="qqSimulationTime">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Simulation time</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Pulsation frequency [Hz]</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qqFrequency">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Pulsation phase</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qqPhase">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Pulsation ratio</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qqRatio">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="qqOkCancel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>115</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>qqEnablePulsation</tabstop>
  <tabstop>qqRealTime</tabstop>
  <tabstop>qqSimulationTime</tabstop>
  <tabstop>qqFrequency</tabstop>
  <tabstop>qqPhase</tabstop>
  <tabstop>qqRatio</tabstop>
  <tabstop>qqOkCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
